package.json

```diff
@@ -92,6 +92,7 @@
     "eslint-plugin-ember": "6.1.0",
     "eslint-plugin-node": "8.0.1",
     "eslint-plugin-prettier": "2.6.0",
+    "graphql-cli": "3.0.11",
```

Comment:
Where do you use this? Or why is this needed?

Reply:
This is needed to run `graphql schema-update` - i forgot to migrate this dependency from ember-caluma-utils.

Reply:
Ok, thanks for the explanation :+1: